sound/soc/intel/boards/Makefile

```diff
@@ -36,7 +36,7 @@ snd-soc-ehl-rt5660-objs := ehl_rt5660.o hda_dsp_common.o
 snd-soc-sof-sdw-objs += sof_sdw.o				\
 			sof_sdw_rt711.o sof_sdw_rt700.o		\
 			sof_sdw_rt1308.o sof_sdw_rt715.o	\
-			sof_sdw_rt5682.o			\
+			sof_sdw_max98373.o sof_sdw_rt5682.o	\
```

Comment:
put on separate line to avoid mixing realtek and maxim :-)

Reply:
Agreed, shall update